### ISSUE TITLE
Convert Routes to use Constants

### ISSUE THREAD

**Issue by saivineeth100** [zerodha/dotnetkiteconnect#40]
Existing
```csharp
private readonly Dictionary<string, string> _routes = new Dictionary<string, string>
{
    ["parameters"] = "/parameters",
    
}
```
Suggested 
Add to Constants.cs or Add new File Paths.cs
```csharp
public static class Paths
{

}
```

**Comment by ajinasokan** [comment]
Thanks for the suggestion. But this will be quite a significant change. Will consider this for the next major version. I will keep the issue open.

**Comment by saivineeth100** [comment]
> Thanks for the suggestion. But this will be quite a significant change. Will consider this for the next major version. I will keep the issue open.

Can I contribute to this or any suggestions via pull requests?

**Comment by ajinasokan** [comment]
I actually have the patch ready. But just not merged yet. Since this is changing code related to all APIs, it will have to be tested as a whole and that is time consuming. That is why I'm delaying it till a major version or a refactor.

**Comment by saivineeth100** [comment]
I am asking in general, can outsiders contribute to Zerodha repos?

**Comment by ajinasokan** [comment]
Yeah sure. Just keep in mind, if it is a large patch then make sure to create an issue with your suggestion and seek maintainers opinion.

**Comment by ajinasokan** [comment]
This is part of the newer versions of lib
